src/thunderbots/software/ai/hl/stp/evaluation/baller.h

```diff
@@ -0,0 +1,19 @@
+#include "ai/world/ball.h"
+#include "ai/world/field.h"
+#include "ai/world/team.h"
+
+namespace Evaluation
+{
+    /**
+     * Returns the robot that either has the ball, or is the closest to having it (and
+     * therefore has the most "presence" over the ball)
+     *
+     * @param team The team to get the baller from
+     * @param ball the Ball
+     * @param field The Field being played on
+     * @return the robot that either has the ball, or is the closest to having it. If the
+     * team has no robots, std::nullopt is returned
+     */
+    std::optional<Robot> getTeamBaller(const Team& team, const Ball& ball,
```

Comment:
I'm not a huge fan of the term "Baller", as I don't think it's very clear. Maybe `getRobotWithEffectiveBallPossession`?

Reply:
renamed